client/a8c-for-agencies/sections/referrals/referral-details/components/assigned-to.tsx

```diff
@@ -81,15 +88,103 @@ const AssignedTo = ( { purchase, handleAssignToSite, data, isFetching }: Props )
 		);
 	}
 
+	let tooltip = '';
+
+	let cancellationInfo = null;
+
+	// If the purchase has a subscription, get details and check the status.
+	const subscription = purchase?.subscription;
+
+	if ( subscription && subscription.status === 'active' && ! subscription.is_auto_renew_enabled ) {
+		// If the subscription is still active but auto-renew is disabled, show the expiration info.
+		cancellationInfo = (
+			<>
+				<p>
+					{ translate(
+						'This product was cancelled, but it will remain active until {{b}}%(expiryDate)s{{/b}}. After that, it will not renew.',
+						{
+							args: {
+								expiryDate: isFetchingProductInfo ? '...' : expiryDate || 'N/A',
+							},
+							components: {
+								b: <b />,
+							},
+							comment: '%(expiryDate)s is the date when the product will expire.',
+						}
+					) }
+				</p>
+				<p>
+					{ translate( '{{link}}Learn more about cancelations ↗{{/link}}', {
+						components: {
+							link: (
+								<a
+									href={ localizeUrl(
+										'https://wordpress.com/support/manage-purchases/cancel-a-purchase/'
+									) }
+									target="_blank"
+									rel="noreferrer noopener"
+								>
+									{  }
+								</a>
+							),
+						},
+					} ) }
+				</p>
+			</>
+		);
+	}
+
+	const isAwaitingPayment = purchase.status === 'pending';
+
+	if ( isAwaitingPayment ) {
+		tooltip = isWPCOMLicense
```

Comment:
Something we should probably improve in the future is how we now have 2 different types of tooltips. They have different styles and one shows onhover while the other onclick. But that's an improvement for another day.
<img width="616" height="219" alt="Screenshot 2025-09-26 at 4 53 10 PM" src="https://github.com/user-attachments/assets/0e9e5213-1392-421c-841d-291817e261a9" />
<img width="545" height="149" alt="Screenshot 2025-09-26 at 4 50 07 PM" src="https://github.com/user-attachments/assets/a407168e-ba27-42f6-a877-f9393fad18c8" />